backend/simple_unified_startup.py

```
@@ -0,0 +1,86 @@
+"""
+Simple Unified JyotiFlow.ai Startup System
+Clean, minimal database initialization with shared pool architecture
+"""
+
+import os
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)